app/validators/claim/litigator_hardship_claim_sub_model_validator.rb

```diff
@@ -0,0 +1,24 @@
+class Claim::LitigatorHardshipClaimSubModelValidator < Claim::BaseClaimSubModelValidator
+  def has_one_association_names_for_steps
+    {
+      case_details: [],
+      defendants: [],
+      offence_details: [],
+      # fixed_fees: [{ name: :fixed_fee }],
+      # graduated_fees: [{ name: :graduated_fee }],
```

Comment:
can be remove, i am guessing!?

Reply:
Yep, grad fees can definitely go, but I've had a chat with the BA and clarified the actual steps needed in this flow.  I'm going through now and deleting a lot of stuff! :)